uk_bin_collection/uk_bin_collection/councils/BedfordshireCouncil.py

```
@@ -1,73 +1,69 @@
         user_postcode = kwargs.get("postcode")
-
+        
         check_uprn(user_uprn)
         check_postcode(user_postcode)
 
-        # Start a new session to walk through the form
+        api_url = f"https://www.centralbedfordshire.gov.uk/waste-and-recycling/waste-collection-schedule/view/{user_uprn}"
+                 
         requests.packages.urllib3.disable_warnings()
-        s = requests.Session()
+        response = requests.get(api_url, verify=False)
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)